GOVERNANCE.md

```diff
@@ -2,94 +2,174 @@
+The Maintainer role is critical to the long-term health of Astro. Maintainers act as the first line of defense when it comes to new issues, pull requests and #support channel activity. Maintainers are most likely the first people that a user will interact with on Discord or GitHub.
 
-**Responsibilities:**
+**A Maintainer is not required to write code!** Some Maintainers spend most of their time inside of Discord, maintaining a healthy community there. Maintainers can also be thought of as **Moderators** on Discord and carry special privileges for moderation.
+
+There is no strict minimum number of contributions needed to reach this level, as long as you can show **sustained** involvement over some amount of time (at least a couple of weeks).
+
+#### Recognized Contributions:
+
+- **GitHub:** Submitting non-trivial pull requests and RFCs
+- **GitHub:** Reviewing non-trivial pull requests and RFCs
+- **Discord:** Supporting users in Discord, especially in the #support channel
+- **Discord:** Active participation in RFC calls and other events
+- **GitHub + Discord:** Triaging and confirming user issues
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privledges:
+
+- All of the privileges of L1, including...
+- `@maintainer` role on [Discord](https://astro.build/chat)
+- New name color on Discord: **blue**.
+- Invitation to the private #maintainers channel on Discord.
+- Invitation to the `maintainers` team on GitHub.
+- Ability to moderate Discord.
+- Ability to push branches to the repo (No more personal fork needed).
+- Ability to review GitHub PRs.
+- Ability to merge _some_ GitHub PRs.
+- Ability to vote on _some_ initiatives (see [Voting](#voting) below).
+
+#### Responsibilities:
+
+- Participate in the project as a team player.
+- Bring a friendly, welcoming voice to the Astro community.
+- Be active on Discord, especially in the #support channel.
+- Triage new issues.
+- Review pull requests.
+- Merge some, non-trivial community pull requests.
+- Merge your own pull requests (once reviewed and approved).
+
+#### Nomination - How to:
 
-- May request write access to relevant Astro projects.
-- GitHub: May work on public branches of the source repository and submit pull requests from that branch to the main branch.
-- GitHub: Must submit pull requests for all changes, and have their work reviewed by other members before acceptance into the repository.
-- GitHub: May merge some pull requests (see Managing Pull Requests)
+To be nominated, a nominee is expected to already be performing some of the responsibilities of a Maintainer over the course of a couple weeks. In the past, we have used "10 PRs" as a rough minimum for potential Maintainers, but there is no hard requirement.
 
-**Nomination:**
+In some rare cases, membership may be revoked by a project Steward. However, under normal circumstances this role is granted for as long as the contributor wishes to engage with the project. 
```

Comment:
This is the first time I see `membership` used like this, I think you mean to remove them as a contributor (level status), so maybe:

```suggestion
In some rare cases, contributors may be revoked by a project Steward. However, under normal circumstances this role is granted for as long as the contributor wishes to engage with the project. 
```